api/envoy/config/filter/http/adaptive_concurrency/v2alpha/adaptive_concurrency.proto

```diff
@@ -61,4 +61,7 @@ message AdaptiveConcurrency {
     GradientControllerConfig gradient_controller_config = 1
         [(validate.rules).message.required = true];
   }
+
+  // If set to true, the adaptive concurrency filter will operate as a pass-through filter.
+  bool disabled = 2;
```

Comment:
I wonder if this should be a field on https://github.com/envoyproxy/envoy/blob/master/api/envoy/api/v2/listener/listener.proto#L24? I.e. this really should just apply to any filter and allow for temporary disabling.

Reply:
This has been asked for previously to be general, so I agree that we should do this at the listener/filter/per-route config level. cc @lizan @kyessenov @rshriram who I think have asked for this previously.

With that said, I don't think a top level `disable` flag adds any real value as the user can just not include the config at all. I assume we actually want this to be a runtime configurable disable. If so can we use `RuntimeUInt32` or potentially define a new `RuntimeFeatureFlag` message that covers this case (and internally allows the key to be specified)?